terraform/app.tf

```diff
@@ -0,0 +1,138 @@
+resource "azurerm_service_plan" "service-plan" {
+  name                = "getanid-authserver-${var.environment_name}-app-spln"
+  location            = data.azurerm_resource_group.group.location
+  resource_group_name = data.azurerm_resource_group.group.name
+  os_type             = "Linux"
+  sku_name            = var.app_service_plan_sku
+
+  lifecycle {
+    ignore_changes = [
+      tags
+    ]
+  }
+}
+
+resource "azurerm_linux_web_app" "auth-server-app" {
+  name                = local.get_an_identity_app_name
+  location            = data.azurerm_resource_group.group.location
+  resource_group_name = data.azurerm_resource_group.group.name
+  service_plan_id     = azurerm_service_plan.service-plan.id
+  https_only          = true
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  site_config {
+    http2_enabled       = true
+    minimum_tls_version = "1.2"
+  }
+
+  app_settings = {
+    HOSTING_ENVIRONMENT                    = var.environment_name,
```

Comment:
Since this is the same map as the other app, can we create a local variable instead?